code/style.css

```diff
@@ -1,4 +1,293 @@
+/* GOOGLE FONTS GOES HERE */ 
+@import url('https://fonts.googleapis.com/css2?family=Bungee&display=swap');
+
+/* GLOBAL STYLES*/ 
+* {padding: 0; margin: 0;}
+
 html {
-  background: #0025ff;
-  color: #fff;
+    background: #222222;
+    color: #E5E5E5;
+    font-family: 'Bungee'; 
+}
+
+body {
+  position:relative;
+  min-height: 100vh;
+}
+
+p {
+  font-family: 'Roboto';
+  font-size: 14px;
+}
+
+/* HEADER & NAVIGATION */
+.flex-container  {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  margin-bottom: 1rem;
+}
+
+.logo {
+    margin: 0 2em;
+}
+
+.navigation  {
```

Comment:
**Desktop**

Links in navbar need a little bit more space from right margin. The word PC is very close to outer margin.  Maybe add as much margin like on left side, by the logo?

**Potential solution:**  Add margin-right : XX px

Reply:
Added some margin-right: 30px; looks a lot better. The logo size begin from the border of the browser and there is no margin there because of img size(img itself has some space between content and border.